Makefile

```diff
@@ -1,5 +1,13 @@
-CFLAGS=-W -Wall -g -O2
 CC=gcc
+CFLAGS=-g -O2 -std=c11 -Werror -Wall -Wextra -pedantic -Wduplicated-cond -Wduplicated-branches -Wlogical-op -Wnull-dereference -Wdouble-promotion  -Wshadow  -Wformat=2 -Wsign-conversion
+ifdef CHKSTAT_TEST
+# for testing, add sanitizers:
+CFLAGS+=-fsanitize=address -fsanitize=pointer-compare -fsanitize=pointer-subtract -fsanitize=undefined
+endif
+# link statically against libstdc++. since some people are afraid of ABI
```

Comment:
For some reason this libstdc++ comment ended up in this commit. This would be confusing, can you remove it?

Reply:
Yes, done